tasks/task03/ Word_stats.py

```diff
@@ -0,0 +1,37 @@
+import re
+print("Welcome to Word Statistic Tool!")
```

Comment:
in this task you do not need to read the strings from the console.
assume you have a list of strings defined in the top like:
```python
texts = [
   "Hello, World!",
   "The world is mine",
   "Hello, how are you?"
]
```

Reply:
Ok
will fix it